pkg/fakerp/fakerp.go

```diff
@@ -187,14 +187,11 @@ func createOrUpdateWrapper(ctx context.Context, p api.Plugin, log *logrus.Entry,
 		return nil, err
 	}
 
-	// TODO: Remove this when APIVersion support lands into fakeRP
-	// and we have more consistatent way to set it
-	// Currently this code part decides if we should deploy
-	// PrivateLinkService and and internal-LB
-	if cs.Properties.NetworkProfile.ManagementSubnetCIDR != nil {
-		cs.Properties.PrivateAPIServer = true
+	// if it's an update, leave it alone, else it's for upgrading to v11 and we
+	// want it off.
+	if !isUpdate {
```

Comment:
why is this code added? in what circumstances in v10 do you run through here such that this does/should trigger?  Can't this PR be a straight code delete?

Reply:
well, there is an update test from v10.0 to top of branch. in that case it will be created with privateAPIServer=true and refuse to remove it. So I was thinking of just setting private to off for the create case (when we are testing v10 to master)